kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RollingOperationContext.java

```diff
@@ -1,112 +0,0 @@
-/**
- * Copyright (C) 2015 Red Hat, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.fabric8.kubernetes.client.dsl.internal;
-
-import java.util.concurrent.TimeUnit;
-
-public class RollingOperationContext {
-
-  private final PodOperationContext podOperationContext;
-  private boolean rolling;
-  private long rollingTimeout;
-  private TimeUnit rollingTimeUnit;
-
-  public RollingOperationContext() {
-    this.podOperationContext = new PodOperationContext();
-  }
-
-  public RollingOperationContext(PodOperationContext podOperationContext, boolean rolling, long rollingTimeout,
-      TimeUnit rollingTimeUnit) {
-    this.podOperationContext = podOperationContext;
-    this.rolling = rolling;
-    this.rollingTimeout = rollingTimeout;
-    this.rollingTimeUnit = rollingTimeUnit != null ? rollingTimeUnit : TimeUnit.MILLISECONDS;
-  }
-
-  public PodOperationContext getPodOperationContext() {
-    return podOperationContext;
-  }
-
-  public Boolean getRolling() {
-    return rolling;
-  }
-
-  public long getRollingTimeout() {
-    return rollingTimeout;
-  }
-
-  public TimeUnit getRollingTimeUnit() {
-    return rollingTimeUnit;
-  }
-
-  public RollingOperationContext withRolling(boolean rolling) {
-    return new RollingOperationContext(podOperationContext, rolling, rollingTimeout, rollingTimeUnit);
-  }
-
-  public RollingOperationContext withRollingTimeout(long rollingTimeout) {
-    return new RollingOperationContext(podOperationContext, rolling, rollingTimeout, rollingTimeUnit);
-  }
-
-  public RollingOperationContext withRollingTimeUnit(TimeUnit rollingTimeUnit) {
-    return new RollingOperationContext(podOperationContext, rolling, rollingTimeout, rollingTimeUnit);
-  }
```

Comment:
how are these options being set now?

Reply:
nevermind, just saw the new lombok builder usage at RollableScalableResourceOperation